packages/cli/src/models/network/TransactionController.ts

```diff
@@ -13,9 +17,18 @@ interface ERC20TokenInfo {
 
 export default class TransactionController {
   public txParams: TxParams;
+  public packageFile: ZosPackageFile;
+  public networkFile: ZosNetworkFile;
 
-  constructor(txParams?: any) {
+  constructor(txParams?: any, network?: string, networkFile?: ZosNetworkFile) {
```

Comment:
txParams should have type TxParams. I've recently added it. Let's keep using it everywhere!